web/src/components/tasks/business-formation/getErrorStateForField.test.ts

```diff
@@ -23,6 +27,39 @@ import {
 const Config = getMergedConfig();
 
 describe("getErrorStateForField", () => {
+  const onlyHasErrorIfUndefinedTest: FormationFields[] = [
```

Comment:
pebble - if we are exporting the two array's from `getErrorStateForField`, why are we creating them here again? It doesn't make sense to have duplicate logic

Reply:
Im some of the test you are referencing the imported arrays, in other tests, you are referencing the arrays you've created here